nixos/modules/services/web-apps/funkwhale.nix

```diff
@@ -0,0 +1,565 @@
+          serviceConfig = {
+            User = "postgres";
+          };
+          script = ''
+            ${pkgs.postgresql}/bin/psql -d funkwhale -c 'CREATE EXTENSION IF NOT EXISTS "unaccent";'
+          '';
+        };
+        funkwhale-init = {
+          description = "Funkwhale initialization";
+          after = [ "funkwhale-psql-init.service" ];
+          wantedBy = [ "funkwhale-server.service" "funkwhale-worker.service" "funkwhale-beat.service" ];
+          before   = [ "funkwhale-server.service" "funkwhale-worker.service" "funkwhale-beat.service" ];
+          environment = funkwhaleEnv;
+          script = ''
+            ${pythonEnv}/bin/python ${pkgs.funkwhale}/manage.py migrate
+            ${pythonEnv}/bin/python ${pkgs.funkwhale}/manage.py collectstatic --no-input
+            if ! test -e ${cfg.dataDir}/createSuperUser.sh; then
+              echo "#!/bin/sh
+            
+              set -a
+              . ${funkwhaleEnvFile}
+              set +a
+              ${pythonEnv}/bin/python ${pkgs.funkwhale}/manage.py createsuperuser" > ${cfg.dataDir}/createSuperUser.sh
+              chmod u+x ${cfg.dataDir}/createSuperUser.sh
+              chown -R funkwhale.funkwhale ${cfg.dataDir}
+            fi
+            if ! test -e ${cfg.dataDir}/config; then
+              mkdir -p ${cfg.dataDir}/config
+              ln -s ${funkwhaleEnvFile} ${cfg.dataDir}/config/.env
+              ln -s ${funkwhaleEnvFile} ${cfg.dataDir}/.env
+            fi
+          '';
+        };
+
+        funkwhale-server = {
+          description = "Funkwhale application server";
+          partOf = [ "funkwhale.target" ];
+
+          serviceConfig = serviceConfig;
+          environment = funkwhaleEnv;
+          script = "${pythonEnv}/bin/daphne -b ${cfg.apiIp} -p ${toString cfg.apiPort} config.asgi:application --proxy-headers";
+
+          wantedBy = [ "multi-user.target" ];
+        };
+
+        funkwhale-worker = {
+          description = "Funkwhale celery worker";
+          partOf = [ "funkwhale.target" ];
+
+          serviceConfig = serviceConfig // { RuntimeDirectory = "funkwhaleworker"; };
+          environment = funkwhaleEnv;
+            
+          script = '' ${pythonEnv}/bin/celery -A funkwhale_api.taskapp worker -l INFO '';
+          wantedBy = [ "multi-user.target" ];
+        };
+
+        funkwhale-beat = {
+          description = "Funkwhale celery beat process";
+          partOf = [ "funkwhale.target" ];
+
+          serviceConfig = serviceConfig // { RuntimeDirectory = "funkwhalebeat"; };
+          environment = funkwhaleEnv;
+          script = ''
+            ${pythonEnv}/bin/celery -A funkwhale_api.taskapp beat -l INFO --schedule="/run/funkwhalebeat/celerybeat-schedule.db"  --pidfile="/run/funkwhalebeat/celerybeat.pid"
+            '';
+
+          wantedBy = [ "multi-user.target" ];
+        };
+
+      };
+
+      environment.systemPackages = with pkgs; [ ffmpeg libjpeg ];
```

Comment:
Why this?

Reply:
These dependencies don't seem to be required anymore, so i've removed this line.
(ffmpeg was used to stream music, libjpeg to manage albums covers. Should  I have required them in the funkwhale package instead of the module ?)